### PR TITLE
[dem] Unifying initial app banner in a single sentence

### PR DESCRIPTION
**Pull request by maceligueta**
Just a minor change

### CODE REVIEW

File: applications/DEMApplication/DEM_application.cpp [hunk 1]
```diff
@@ -482,15 +482,13 @@ template class DEM_compound_constitutive_law<DEM_D_Linear_viscous_Coulomb, DEM_D
...
-    KRATOS_INFO("DEM") << "Importing DEMApplication... ";
-
-    KRATOS_INFO("") << "( compiled in mode \"" << Kernel::BuildType() << "\" )";
+    KRATOS_INFO("") << "\n"
+                    << "     KRATOS |  _ \\| ____|  \\/  |  _ \\ __ _  ___| | __      \n"
+                    << "            | | | |  _| | |\\/| | |_) / _` |/ __| |/ /      \n"
+                    << "            | |_| | |___| |  | |  __/ (_| | (__|   <       \n"
+                    << "            |____/|_____|_|  |_|_|   \\__,_|\\___|_|\\_\\      \n"
+                    << "Importing DEMApplication "
+                    << "( compiled in mode \"" << Kernel::BuildType() << "\" ) ...";
```
Comment (philbucher) [inline applications/DEMApplication/DEM_application.cpp]:
```suggestion
```

this is already printed in the Kratos-logo

**Review by philbucher — COMMENTED** [review]
this satisfies my code-ocd :D

**Comment by philbucher** [comment]
btw why is it called DEMPack and not just DEM?

**Comment by maceligueta** [comment]
> btw why is it called DEMPack and not just DEM?

Politics...

**Review by salvalatorre — APPROVED** [review]
Good!

**Comment by maceligueta** [comment]
> this satisfies my code-ocd :D

I had to Google ocd XD

File: applications/DEMApplication/DEM_application.cpp (hunk 1 above)
Comment (maceligueta) [reply applications/DEMApplication/DEM_application.cpp]: Semi colon missing, so I did it in a different commit

**Pull request merged**